lib/goals/EcsDeploy.ts

```diff
@@ -105,19 +108,20 @@ export interface EcsDeployment extends Deployment {
 }
 
 // tslint:disable-next-line:max-classes-per-file
-export class EcsDeployer implements Deployer<EcsDeploymentInfo, EcsDeployment> {
+export class EcsDeployer implements Deployer<EcsDeploymentInfo & EcsDeployRegistration, EcsDeployment> {
     constructor(private readonly projectLoader: ProjectLoader) {
     }
 
     public async deploy(da: DeployableArtifact,
                         esi: EcsDeploymentInfo,
                         log: ProgressLog,
                         credentials: ProjectOperationCredentials): Promise<EcsDeployment[]> {
-        logger.info("Deploying app [%j] to ECS [%s]", da, esi.description);
+        log.write(`Deploying service [${da.name}] to ECS [${esi.cluster}]`);
```

Comment:
We don't use [] in progressLog messages. I'd change that to `Deploying service '${da.name}' to ECS '${esi.cluster}'?

Reply:
Ah, np.  Updating.